deeppavlov/models/preprocessors/torch_transformers_preprocessor.py

```diff
@@ -202,6 +203,7 @@ class TorchSquadTransformersPreprocessor(Component):
 
     def __init__(self,
                  vocab_file: str,
+                 lang: str,
```

Comment:
add to docstring and change type of the variable to `Literal['en', 'ru']`
Literal should be imported from the typing_extensions:
```
from typing_extensions import Literal
```